File: csL2/Domain/Inscriere.cs
namespace lab2_mpp.domain;

public class Inscriere : Entity<int>
{
    public String ParticipantName { get; set; }
    public int ParticipantAge { get; set; }
    public Proba Proba { get; set; }

    public Inscriere(String participantName, int participantAge, Proba proba)
    {
        ParticipantName = participantName;
        ParticipantAge = participantAge;
        Proba = proba;
    }
    
    public override string ToString()
    =>$"Inscriere {{ name: {ParticipantName}; age: {ParticipantAge}; proba: {Proba} }}";
    
}

File: javaL2/src/main/java/ro/mpp2024/domain/Inscriere.java
package ro.mpp2024.domain;

public class Inscriere extends Entity<Integer>{

    private String participantName;
    private int participantAge;

    private Proba proba;
    public Proba getProba() {
        return proba;
    }
    public void setProba(Proba p) {
        this.proba = p;
    }
    public Inscriere(String participantName, int participantAge) {
        this.participantName = participantName;
        this.participantAge = participantAge;
    }

    public String getParticipantName() {
        return participantName;
    }
    public void setParticipantName(String participantName) {
        this.participantName = participantName;
    }
    public int getParticipantAge() {
        return participantAge;
    }
    public void setParticipantAge(int participantAge) {
        this.participantAge = participantAge;
    }

}
